yarn-project/end-to-end/scripts/native-network/boot-node.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
+set -eu
+
+# Get the name of the script without the path and extension
+SCRIPT_NAME=$(basename "$0" .sh)
+
+# Redirect stdout and stderr to <script_name>.log while also printing to the console
+exec > >(tee -a "$(dirname $0)/logs/${SCRIPT_NAME}.log") 2> >(tee -a "$(dirname $0)/logs/${SCRIPT_NAME}.log" >&2)
+
+# Starts the Boot Node
+
+# Set environment variables
+export PORT="8080"
+export LOG_LEVEL="debug"
+export LOG_JSON="1"
+export DEBUG="aztec:*,-aztec:avm_simulator:*"
+export ETHEREUM_HOST="http://127.0.0.1:8545"
+export P2P_ENABLED="true"
+export VALIDATOR_DISABLED="true"
+export SEQ_MAX_SECONDS_BETWEEN_BLOCKS="0"
+export SEQ_MIN_TX_PER_BLOCK="1"
+export P2P_TCP_ANNOUNCE_ADDR="0.0.0.0:40400"
```

Comment:
I recall in the past i had issues where the announce addresses was not `127.0.0.1`. But I cannot recall why - writing this just incase